README.md

```diff
@@ -97,6 +97,7 @@ Add a vim-plug section to your `~/.vimrc` (or `stdpath('config') . '/init.vim'`
 
 ```vim
 " Specify a directory for plugins
+" - For Windows: '~/vimfiles/plugged'
```

Comment:
This doesn't apply to Neovim on Windows I guess? If that is the case, we should put this line *after* `For Neovim`.

Reply:
That's right, it doesn't apply to Neovim on Windows.

Neovim handles this problem in a much more sensible way, since it has `stdpath('data')` to generate the correct path for all platforms. The existing directions for Neovim are correct for Linux/macOS/Windows.

So with this change, the order listed in the README would be:
Vim (Linux/macOS)
Vim (Windows)
Neovim (all platforms)

I'm happy to rearrange the order or make the text more explicit if that would be preferred.

Reply:
I've pushed a new commit consistent with my comment above.